src/Nox.Integration/Exceptions/NoxIntegrationTypeConversionException.cs

```diff
@@ -0,0 +1,13 @@
+namespace Nox.Integration.Exceptions;
+
+[Serializable]
```

Comment:
deserialization constructor is mandatory for [Serializable]

Reply:
Nice, thankx